api/portainer.go

```diff
@@ -215,6 +215,12 @@ type (
 	// EndpointType represents the type of an endpoint
 	EndpointType int
 
+	// EndpointRelation represnts a endpoint relation object
+	EndpointRelation struct {
```

Comment:
I'm still unsure about the name EndpointRelation as it might be too generic, since this is supposed to represents the relation between an endpoint and Edge stacks what about `EndpointToEdgeStackRelation` ?

Reply:
felt like it was very long (`EndpointToEdgeStackRelationService`) and that we might want in the future to add other relations to this object instead of adding another resource, but I'll change it

Reply:
It is ok to keep it as EndpointRelation for future evolution.